hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java

```diff
@@ -311,8 +311,8 @@ public static InetSocketAddress getReconAddresses(
   public static InetSocketAddress getSingleSCMAddress(
       ConfigurationSource conf) {
     Collection<InetSocketAddress> singleton = getSCMAddresses(conf);
-    Preconditions.checkArgument(singleton.size() == 1,
-        MULTIPLE_SCM_NOT_YET_SUPPORTED);
+   // Preconditions.checkArgument(singleton.size() == 1,
```

Comment:
Can we remove this?

Reply:
Thanks @vivekratnavel . We will remove this once the the proper SCM address validation patch is in . Its a work in progress.